src/store/plugins/geolocation-management.plugin.js

```diff
@@ -22,6 +22,11 @@ function setCenterIfInBounds(store, center) {
                 center: center,
                 ...dispatcher,
             })
+
+            if (firstTimeActivatingGeolocation) {
+                firstTimeActivatingGeolocation = !firstTimeActivatingGeolocation
+                store.dispatch('setZoom', { zoom: 8, ...dispatcher })
```

Comment:
please use `store.state.position.projection.get1_25000ZoomLevel()` instead of a magic 8 number